docs/IML_Contributor_Workflow.md

```diff
@@ -0,0 +1,129 @@
+[**Table of Contents**](index.md)
+
+![workflow](md_Graphics/workflow_sm.jpg)
+
+# IML Contributor Workflow
+## Overview
```

Comment:
Change unit test to automated / manual tests in the diagram

Reply:
changed